build/ci/linux/build.sh

```diff
@@ -68,6 +68,13 @@ echo "=== ENVIRONMENT === "
 cat $BUILD_TOOLS/environment.sh
 source $BUILD_TOOLS/environment.sh
 
+# disable crashpad client and update module on aarch64 and armv7l due to no prebuilts being available
+if [ "$PACKARCH" == "aarch64" ] || [ "$PACKARCH" == "armv7l" ]; then
```

Comment:
Where does the `PACKARCH` variable come from here?

Reply:
It is explicitly passed in build/ci/linux_ARM/setup.sh (and in build/ci/tools/osuosl/publish.sh) and it is stored in $BUILD_TOOLS/environment.sh (so that the following scripts can load it from that auto-generated script).

Reply:
It is better to explicitly pass it to all scripts where it is needed, like other options. I would not like this option to be different from the others. For example, in a setup, only "not our" variables are written to environment variables, but those that are required for Qt.
It is difficult to use and improvement scripts when variables are passed in different ways, some explicitly, some as environment variables